assets/js/default-calendar.js

```diff
@@ -405,7 +405,7 @@ jQuery(function ($) {
 		window.print();
 
 		$('body > .simcal-calendar').remove();
-		$('body').children().show();
+		$('body').children().not('script').show();
```

Comment:
What is this for?

Reply:
in js display block it was added display block to script tag when closing print screen, and it added some content on bottom, so I have added a condition in it. to skip the script tag.